### DISCUSSION TITLE
Maybe create Discord server?

### DISCUSSION THREAD

**Discussion by evilenzo** [vsg-dev/VulkanSceneGraph#1009] [Q&A] [answered]
Github Q&A looks overcomplicated for newbies imo, what do you think about creation of Discord server?

**Comment by robertosfield** [comment, marked as answer]
Distributing support across multiple channels spread the places where information can be found and increases how much time it takes to support the community.  These are both bad things that would hamper the project as whole, even if it would appease some folks personal preference in how they want their free support.
